nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -257,20 +259,27 @@ in
       fonts.fonts = with pkgs; [
         cantarell-fonts
         dejavu_fonts
+        noto-fonts-emoji
```

Comment:
Could we make a `emojiFont` option until the [conflict](https://github.com/NixOS/nixpkgs/pull/67014#issuecomment-522839053) is resolved? I suspect it is caused by JoyPixels missing from, will test the hypothesis.